src/Operation/Watch.php

```diff
@@ -70,38 +70,29 @@ class Watch implements Executable, /* @internal */ CommandSubscriber
 
     private const WIRE_VERSION_FOR_START_AT_OPERATION_TIME = 7;
 
-    /** @var Aggregate */
-    private $aggregate;
+    private Aggregate $aggregate;
 
     /** @var array */
     private $aggregateOptions;
 
     /** @var array */
     private $changeStreamOptions;
```

Comment:
These are missing an `array` type.

Reply:
Correct. I've added a type hint to avoid the cast.